src/olympia/devhub/templates/devhub/new-landing/components/connect.html

```diff
@@ -7,8 +7,8 @@ <h2>{{ _('Connect with us') }}</h2>
       <div class="DevHub-Connect-section">
         <h4>{{ _('Twitter') }}</h4>
         <ul class="DevHub-content-copy--Connect-twitter-list">
-          <li>{{ _('For developers') }}:<a class="Before-Icon Before-Icon-twitter" href="https://twitter.com/mozamo" target="_blank" rel="noopener noreferrer">@mozamo</a></li>
-          <li>{{ _('For end users') }}:<a class="Before-Icon Before-Icon-twitter" href="https://twitter.com/rockyourfirefox" target="_blank" rel="noopener noreferrer">@rockyourfirefox</a></li>
+          <li>{{ _('For developers') }}:<a class="Before-Icon Before-Icon-twitter" href="https://twitter.com/mozamo" target="_self" rel="noopener noreferrer">@mozamo</a></li>
```

Comment:
Instead of changing the `target` attribute, you can simply remove it along with the `rel` attribute as per https://github.com/mozilla/addons-server/issues/12701#issuecomment-547977053

Reply:
Got it!